### PR TITLE
create the org.apache.cxf.cxf.rt.bindings.xml.2.6 bundle 

### PR DESCRIPTION
**Pull request by WhiteCat22**
Create the org.apache.cxf.cxf.rt.bindings.xml.2.6 bundle to bring in the changes we made to Apache CXF 2.6.2 for jaxws-2.2

### CODE REVIEW

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.2.6/.settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -0,0 +1,280 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.codegen.unusedLocal=preserve
+org.eclipse.jdt.core.compiler.compliance=1.8
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.2.6/.settings/org.eclipse.jdt.core.prefs]: This needs to match the java version that is used in the current JAX-WS bundles. Should be 1.7

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.2.6/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-bindings-xml;2.6.2}}!/META-INF/MANIFEST.MF,bnd.overrides
+
+javac.source: 1.8
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.2.6/bnd.bnd]: 1.7 to match versions

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.2.6/bnd.overrides [hunk 3]
```diff
@@ -0,0 +1,13 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.2.6
+
+Bundle-Activator: com.ibm.ws.jaxws22.cxf.rt.bindings.xml.NoOpActivator
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.2.6/bnd.overrides]: 1.7

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build
previous build failed with network issues.

**Pull request merged**